### PR TITLE
Add an SDK-Version header to each request

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: Runtime/Game/Resources/LootLockerConfig.cs [hunk 1]
```diff
@@ -78,8 +81,32 @@ static void CreateConfigFile()
...
+            {
+                EditorApplication.update -= ListRequestProgress;
+                foreach (var package in ListInstalledPackagesRequest.Result)
+                {
+                    if (package.name.Equals("com.lootlocker.lootlockersdk"))
+                    {
+                        LootLockerConfig.current.sdk_version = package.version;
+                    }
+                }
+            }
```
Comment (JohannesLoot) [inline Runtime/Game/Resources/LootLockerConfig.cs:106]: You need to unsubscribe from EditorApplication.update. So add a EditorApplication.update -= ListRequestProgress; here.

File: Runtime/Game/Resources/LootLockerConfig.cs [hunk 2]
```diff
@@ -78,8 +81,32 @@ static void CreateConfigFile()
...
+        static void ListRequestProgress()
+        {
+            if (ListInstalledPackagesRequest.IsCompleted)
+            {
+                EditorApplication.update -= ListRequestProgress;
+                foreach (var package in ListInstalledPackagesRequest.Result)
+                {
+                    if (package.name.Equals("com.lootlocker.lootlockersdk"))
+                    {
+                        LootLockerConfig.current.sdk_version = package.version;
```
Comment (JohannesLoot) [inline Runtime/Game/Resources/LootLockerConfig.cs:103]: Add a break here so we don't look through more packages than necessary.

File: Runtime/Game/Resources/LootLockerConfig.cs [hunk 3]
```diff
@@ -1,7 +1,10 @@
 using System;
 using System.IO;
+using UnityEditor.PackageManager.Requests;
```
Comment (JohannesLoot) [inline Runtime/Game/Resources/LootLockerConfig.cs:3]: Move this below #if UNITY_EDITOR

**Review by AlmightyMikkel — APPROVED** [review]
Looks good!

**Pull request merged**